gatsby/content/docs/online-events.mdx

```diff
@@ -19,8 +19,8 @@ recreate the fun and atmosphere of a real meetup in a decentralised way.
 ## Use Jitsi for Video
 
 It's important to us, wherever possible, to use Open tools. Jitsi Meet is Open 
-Source, self-hostable video conferencing platform. [Installing Jitsi is a 
-much-documented experience](https://github.com/jitsi/jitsi-meet/blob/master/doc/
+Source, self-hostable video conferencing platform.
+[Installing Jitsi is a much-documented experience](https://github.com/jitsi/jitsi-meet/blob/master/doc/
```

Comment:
This still has a new line in the middle of the URL (which is the issue, md doesn't really care about having a line break in the middle of the text).

Reply:
Good spot